docroot/themes/humsci/humsci_basic/src/scss/tools/_mixins.fonts.scss

```diff
@@ -0,0 +1,114 @@
+// Started using humsci_basic `hb` prefix here
+// Thoughts?
+@mixin hb_heading-1 {
+  @include colorful {
+    font-weight: theme-font-weight(semibold);
+    font-size: 40px;
```

Comment:
Kasey, I'm curious about whether you think I should use rems here? I still don't quite understand the benefits over pixels

Reply:
Rems for sure!

Reply:
the only thing I didnt change rems for was small amounts of pixels, like borders